webapp/src/ts/components/search-bar/search-bar.component.ts

```diff
@@ -55,9 +59,11 @@ export class SearchBarComponent implements AfterContentInit, AfterViewInit, OnDe
     const subscription = combineLatest(
       this.store.select(Selectors.getSidebarFilter),
       this.store.select(Selectors.getFilters),
-    ).subscribe(([sidebarFilter, filters]) => {
+      this.store.select(Selectors.getSearchBar),
+    ).subscribe(([sidebarFilter, filters, searchBar]) => {
       this.activeFilters = sidebarFilter?.filterCount?.total || 0;
       this.filters = filters;
+      this.openSearch = searchBar.isOpen;
```

Comment:
It's not closing properly in navigation:
https://github.com/user-attachments/assets/40681843-ee3d-4e51-9b40-b2b743e5c001

In the master branch:
https://github.com/user-attachments/assets/fd2c948d-faae-4dd8-a7b0-7dedda579cf7

Reply:
This was caused by the approach to manage search bar using state. I'm thinking of this solution. 

```
ngOnDestroy() {
  if (this.openSearch) {
    this.globalActions.setSearchBar({ isOpen: false });
  }
  this.subscription.unsubscribe();
}
```

The other option is 

```
ngOnDestroy() {
  this.clear();
  this.subscription.unsubscribe();
}
```

But clear() does more than just close the searchBar so I am leaning towards the first

Reply:
It the state was already `false` it won't emit, so you can do this a bit simpler: 
```
ngOnDestroy() {
  this.globalActions.setSearchBar({ isOpen: false });
  this.subscription.unsubscribe();
}
```